spec/controllers/slack_controller_spec.rb

```diff
@@ -0,0 +1,153 @@
+require 'rails_helper'
+
+describe ::DiscourseSlack::SlackController do
+  routes { ::DiscourseSlack::Engine.routes }
```

Comment:
without that line it gives error. I followed this tutorial https://www.relishapp.com/rspec/rspec-rails/docs/routing-specs/engine-routes